antropy/entropy.py

```diff
@@ -464,12 +465,17 @@ def app_entropy(x, order=2, metric="chebyshev"):
         One-dimensional time series of shape (n_times).
     order : int
         Embedding dimension. Default is 2.
+    tolerance : float
+        Tolerance value for acceptance of template vector. Default is 0.2
+        times the standard deviation. 
     metric : str
         Name of the distance metric function used with
         :py:class:`sklearn.neighbors.KDTree`. Default is to use the
         `Chebyshev <https://en.wikipedia.org/wiki/Chebyshev_distance>`_
         distance.
-
+    tolerance_reference : str
+        Switch between tolerance value relative to standard deviation (`SD`)
+        and absolute values (`abs`). Default is `SD`. 
```

Comment:
`abs` is confusing because it implies that you're looking at the absolute values of the signal. I think I would rather let the user calculate the tolerance parameter for themselves, and if not specified, use the default `0.2 * np.std(x, ddof=0)`:

```python
def app_entropy(..., tolerance=None):

  if tolerance is None:
    r = 0.2 * np.std(x, ddof=0)
```

Reply:
In doing so, I would remove the `tolerance_reference` parameter